frontend/src/hooks/api/getters/useInstanceMetrics/useInstanceMetrics.ts

```diff
@@ -18,10 +18,13 @@ export interface IInstanceMetricsResponse {
     error?: Error;
 }
 
-export const useInstanceMetrics = (): IInstanceMetricsResponse => {
+export const useInstanceMetrics = (
+    options: SWRConfiguration = {}
+): IInstanceMetricsResponse => {
     const { data, error, mutate } = useSWR(
         formatApiPath(`api/admin/metrics/rps`),
-        fetcher
+        fetcher,
+        options
```

Comment:
Passing in the options as a parameter allows us more flexibility if needed, like updating this based on an interval in the future, if needed. Not being used yet though, I only experimented a bit with it, so I can clean it up for now and only add it when needed.